tasks/add-fbc-contribution/tests/test-add-fbc-contribution.yaml

```diff
@@ -133,6 +145,26 @@ spec:
                 echo "fbcFragment does not match"
                 exit 1
               fi
+
+              # Check mustPublishIndexImage value (true or false)
+              if [ "$(params.must-publish-index-image)" = "true" ]; then
```

Comment:
I guess here should be `$(params.isFbcOptIn)` and `$(params.mustPublishIndexImage)`

Reply:
Yes, thanks; that was a good catch. I have also updated the mock results, and now everything looks good.